### ISSUE TITLE
Dev Branch break existing code relating to LimitStencils

### ISSUE THREAD

**Issue by oracle3001** [PixarAnimationStudios/OpenSubdiv#992]
This code was valid until the dev branch.

```
auto g_controlStencils = LimitStencilTableFactory::Create(*_refiner, locs);

	//Evaluate the surface with parametric coordinates
	int n_limit_stencils = g_controlStencils->GetNumStencils();
	for (int i = 0; i < n_limit_stencils; ++i)
	{
		LimitStencil limit_st = g_controlStencils->GetLimitStencil(i); 
                unsigned int size_st = limit_st.GetSize();
.....
```

error C2247: 'OpenSubdiv::v3_3_3::Far::StencilReal<float>::GetSize' not accessible because 'OpenSubdiv::v3_3_3::Far::LimitStencil' uses 'private' to inherit from 'OpenSubdiv::v3_3_3::Far::LimitStencilReal<float>'

**Comment by davidgyu** [comment]
Thanks for filing. Yes, we will fix this oversight ASAP. Thanks!

**Comment by jtran56** [comment]
Filed as internal issue #OSD-240.

**Comment by davidgyu** [comment]
Fixed by #993
